plugins/legacy-plugin-chart-heatmap/src/controlPanel.ts

```diff
@@ -204,7 +210,31 @@ const config: ControlPanelConfig = {
               ),
             },
           },
-          'y_axis_format',
+        ],
+        ['y_axis_format'],
+        [
+          {
+            name: 'sort_x_axis',
+            config: {
+              type: 'SelectControl',
+              label: t('Sort X Axis'),
+              choices: sortAxisChoices,
+              clearable: false,
+              default: 'alpha_asc',
```

Comment:
I think we can add `renderTrigger: true` here (it shouldn't affect the query based on my tests)